src/main/java/org/cpsc304/bigdata/db/dao/DiseaseDAOImpl.java

```diff
@@ -77,6 +77,42 @@
         }
     }
 
+    @Override
+    public List<Disease> findDiseaseByPara(final String table, final String number) {
```

Comment:
What does `Para` mean?

Reply:
just mean that users can cross reference disease with either clinical trial or treatment and find the disease with more than X clinical trials or treatments